src/components/Dashboard/StatusPanel.js

```diff
@@ -1,9 +1,12 @@
 import React, { Component } from 'react';
 
+import _ from 'lodash';
```

Comment:
This imports the whole `lodash` library making our bundle size larger. We should import individual operators instead.

```js
import _merge from 'lodash/merge';
import _findIndex from 'lodash/findIndex';
```